testbook/testbook.py

```diff
@@ -4,14 +4,22 @@
 
 
 class testbook:
-    def __init__(self, nb_path, prerun=None):
+    def __init__(self, nb_path, prerun=None, execute=False, timeout=60):
         self.nb_path = nb_path
         self.prerun = prerun
+        self.execute = execute
+        self.timeout = timeout
 
         with open(self.nb_path) as f:
             nb = nbformat.read(f, as_version=4)
 
-        self.client = TestbookNotebookClient(nb)
+        self.client = TestbookNotebookClient(nb, timeout=self.timeout)
+
+    def _prepare(self):
+        if self.execute:
+            self.client.execute()
+        elif self.prerun is not None:
```

Comment:
What if I pass `execute` and `prerun`? I think the behavior would be a bit confusing.